source/aggregation-tutorials/filtered-subset.txt

```diff
@@ -0,0 +1,194 @@
+.. _node-aggregation-filtered-subset:
+
+===============
+Filtered Subset
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, sort, limit, aggregation
+
+Introduction
+------------
+
+In this tutorial, you can learn how to use the {+driver-short+} to
+find a subset of documents in a collection by using aggregation.
```

Comment:
S: also explain that this tutorial shows how to format the output using sorts, setting/unsetting fields, etc

Something like:
```suggestion
In this tutorial, you can learn how to use the {+driver-short+} to
find a subset of documents in a collection and format results by using aggregation.
```